client/app/queue/CavcDetail.jsx

```diff
@@ -0,0 +1,107 @@
+/* eslint-disable no-nested-ternary */
+import React from 'react';
+import PropTypes from 'prop-types';
+import _ from 'lodash';
+
+import { detailListStyling, getDetailField } from './Detail';
+import { DateString } from '../util/DateUtil';
+import BareList from '../components/BareList';
+import COPY from '../../COPY';
+import CAVC_REMAND_SUBTYPE_NAMES from '../../constants/CAVC_REMAND_SUBTYPE_NAMES';
+
+const CavcDetail = (props) => {
+  const {
+    cavc_docket_number: docketNumber,
+    represented_by_attorney: hasAttorney,
+    cavc_judge_full_name: judgeName,
+    cavc_decision_type: procedure,
+    remand_subtype: type,
+    decision_date: decisionDate,
+    judgement_date: judgementDate,
+    mandate_date: mandateDate
+  } = props;
+
+  const details = [];
+
+  if (docketNumber) {
+    details.push({
+      label: COPY.CASE_DETAILS_CAVC_DOCKET_NUMBER,
+      value: docketNumber
+    });
+  }
+
+  if (hasAttorney === true) {
+    details.push({
+      label: COPY.CASE_DETAILS_CAVC_ATTORNEY,
+      value: 'Yes'
+    });
+  } else if (hasAttorney === false) {
+    details.push({
+      label: COPY.CASE_DETAILS_CAVC_ATTORNEY,
+      value: 'No'
+    });
+  }
+
+  if (judgeName) {
+    details.push({
+      label: COPY.CASE_DETAILS_CAVC_JUDGE,
+      value: judgeName
+    });
+  }
+
+  if (procedure) {
+    details.push({
+      label: COPY.CASE_DETAILS_CAVC_PROCEDURE,
+      value: _.startCase(procedure)
+    });
+  }
+
+  if (type && procedure !== 'death_dismissal' && procedure !== 'straight_reversal') {
```

Comment:
Would it be easier to just check for `procedure === CAVC_DECISION_TYPES.remand` here?
```suggestion
  if (type && procedure === CAVC_DECISION_TYPES.remand) {
```